pkg/volume/rbd/rbd.go

```diff
@@ -354,7 +355,18 @@ func (plugin *rbdPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*vol
 	}
 	s := dstrings.Split(sourceName, "-image-")
 	if len(s) != 2 {
-		return nil, fmt.Errorf("sourceName %s wrong, should be pool+\"-image-\"+imageName", sourceName)
+		// Global device mount path doesn't conform to canonical format:
```

Comment:
be more verbose, something like `mounter.GetDeviceNameFromMount` expects `mounts` that is missing in old rbd path

Reply:
Thanks for reminding, I've added more explanations for this scenario, PTAL.